release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -735,6 +735,15 @@ With this release, if the `noProxy` field is set and the route is reachable with
 [id="ocp-4-14-networking"]
 === Networking
 
+[id="ocp-4-14-ipv6-primary-address-family-vsphere-dual-stack-clusters"]
+==== IPv6 as primary IP address family on vSphere dual-stack clusters
+
+During cluster installation on vSphere, you can configure IPv6 as the primary IP address family on a dual-stack cluster. To enable this feature when installing a new cluster, specify an IPv6 address family before an IPv4 address family for the machine network, cluster network, service network, API VIPs, and ingress VIPs.
+
+* Installer-provisioned infrastructure: xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#modifying-install-config-for-dual-stack-network_installing-vsphere-installer-provisioned-network-customizations[Deploying with dual-stack networking]
+* User-provisioned infrastructure: xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installation-configuration-parameters-network_installing-vsphere-installer-provisioned-network-customizations[Network configuration parameters]
```

Comment:
An observation (with very little insight into the technical aspect of the doc): Should the link be reworded to reflect network configuration customizations (and not parameters) based on the link's target? Not sure, but wanted to mention.

Reply:
ahh yeah good catch. The content moved in 4.14 and I used a 4.13 reference (surprised it build honestly).

Thanks